lib/util/stream_utils.js

```diff
@@ -1445,3 +1469,44 @@ shaka.util.StreamUtils.DecodingAttributes = {
   POWER: 'powerEfficient',
   BANDWIDTH: 'bandwidth',
 };
+
+/**
+ * @private {!Set.<string>}
+ */
+shaka.util.StreamUtils.supportedImageMimeTypes = new Set()
+    .add('image/svg+xml')
+    .add('image/png')
+    .add('image/jpeg');
+
+/**
+ * @private {!Set.<string>}
+ */
+shaka.util.StreamUtils.unsupportedImageMimeTypes = new Set();
+
+/**
+ * @const {string}
+ * @private
+ */
+shaka.util.StreamUtils.minWebPImage = 'data:image/webp;base64,UklGRjoAAABXRU' +
+    'JQVlA4IC4AAACyAgCdASoCAAIALmk0mk0iIiIiIgBoSygABc6WWgAA/veff/0PP8bA//LwY' +
+    'AAA';
+
+/**
+ * @const {string}
+ * @private
+ */
+shaka.util.StreamUtils.minAvifImage = 'data:image/avif;base64,AAAAIGZ0eXBhdm' +
+    'lmAAAAAGF2aWZtaWYxbWlhZk1BMUIAAADybWV0YQAAAAAAAAAoaGRscgAAAAAAAAAAcGljd' +
+    'AAAAAAAAAAAAAAAAGxpYmF2aWYAAAAADnBpdG0AAAAAAAEAAAAeaWxvYwAAAABEAAABAAEA' +
+    'AAABAAABGgAAAB0AAAAoaWluZgAAAAAAAQAAABppbmZlAgAAAAABAABhdjAxQ29sb3IAAAA' +
+    'AamlwcnAAAABLaXBjbwAAABRpc3BlAAAAAAAAAAIAAAACAAAAEHBpeGkAAAAAAwgICAAAAA' +
+    'xhdjFDgQ0MAAAAABNjb2xybmNseAACAAIAAYAAAAAXaXBtYQAAAAAAAAABAAEEAQKDBAAAA' +
+    'CVtZGF0EgAKCBgANogQEAwgMg8f8D///8WfhwB8+ErK42A=';
+
+/**
+ * @const {!Map.<string, string>}
+ * @private
+ */
+shaka.util.StreamUtils.minImage = new Map()
```

Comment:
Please end all of these private variable names with an underscore.  Thanks!

Reply:
Done!

Reply:
I think you missed this one.

Reply:
Ups....